apps/website/src/pages/docs/header.mdx

```diff
@@ -14,68 +14,45 @@ import * as AllExamples from 'examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-header' />
-
 <LiveExample src='Header.main.tsx'>
   <AllExamples.HeaderMainExample client:load />
 </LiveExample>
 
-The headers seen when in our CONNECT Portal environment. A header sits at the top of a page and is part of a consistent user experience.
-
-## Main CONNECT Portal header
-
-The CONNECT portal header contains the services menu, home, enterprise, assets and projects links beginning from left to right. From right to left there is the user profile, help, and notification links.
+The first thing most people look at when they arrive at a new web page is the header. It is usually stuck at the top of the page. The header contains vital information about your company's name and function, as well as the purpose of the specific page.
 
-Header with circular callouts, numbered from 1 to 8.
+## Usage
 
-- Services menu
-- Home page
-- Enterprise portal
-- Assets menu
-- Projects menu
-- Notifications
-- Help
-- User icon
+`<Header>` consists of various subcomponents and can be customized by modifying its props.
 
-### Actions
+- Logo can be added using the `appLogo` prop and specifying the desired logo under the `<HeaderLogo>` subcomponent.
+- Breadcrumbs trail can be implemented using the `breadcrumbs` prop and `<HeaderBreadcrumbs>` subcomponent. Individual buttons can be added within the breadcrumbs trail using the `<HeaderButton>` subcomponent. For a dropdown menu, the `menuItems` prop can be used inside the `<HeaderButton>` for showing items and `onClick()` properties for handling actions.
+- Buttons/Icons/Content can be displayed towards the left of menuitems using the `actions` prop.
 
-- Services menu pops up a list of services available to the user.
-- Home page takes you to CONNECT Center.
-- Enterprise Portal takes you to the Subscription Services Portal.
-- Assets menu takes you to the assets page.
-- Projects menu takes you to the ProjectWise Projects page.
-- Notifications pops up your list of notifications.
-- Help takes you to Bentley Communities for Bentley Cloud and Web Services.
-- User profile is a drop down with options for your user profile.
+## Customizations
 
-## Secondary header
+Alter standard header section by adding new custom elements and content.
 
-The secondary header contains the product icon, product name, iModel name and the name of the initially opened view. On the right side of the bar is the settings menu.
+### Full Header
 
-Secondary header with circular callouts, numbered from 1 to 5.
+Default header with additional items.
```

Comment:
As mentioned in https://github.com/iTwin/iTwinUI/pull/1646#discussion_r1371974211, this full header is not a "customization".

suggestions:
1. move the full header example under "Usage" (since it is a good way to demonstrate some of the props and subcomponents)
2. remove the "Customizations" and Full Header" sections
3. promote the "Slim Header" heading from h3 to h2